server.js

```diff
@@ -117,9 +120,11 @@ app.prepare().then(async () => {
   server.all("/_/:a/:b", async (r, s) => handler(r, s, r.url));
 
   server.get("/:username", async (req, res) => {
+    let mobile = Validations.onMobile(req.headers["user-agent"]);
```

Comment:
Probably should be using `~/common/window` for user agents, since Validations is for data integrity, but your call here.